app/controllers/ProjectSidewalkAPIController.scala

```diff
@@ -152,7 +189,7 @@ class ProjectSidewalkAPIController @Inject()(implicit val env: Environment[User,
     */
   def getAccessScoreNeighborhoodsV1(lat1: Double, lng1: Double, lat2: Double, lng2: Double) = UserAwareAction.async { implicit request =>
     apiLogging(request.remoteAddress, request.identity, request.toString)
-    Future.successful(Ok(getAccessScoreNeighborhoodsGeneric(lat1, lng1, lat2, lng2, version = 1, request.toString)))
+    Future.successful(Ok(getAccessScoreNeighborhoodsJson(version = 1, Array(min(lat1, lat2), max(lat1, lat2), min(lng1, lng2), max(lng1, lng2)))))
```

Comment:
Can you make a coordinates variable on its own line so the lines stay below 120 chars?